Formula/licensed.rb

```diff
@@ -0,0 +1,33 @@
+class Licensed < Formula
+  desc "Cache and verify the licenses of dependencies"
+  homepage "https://github.com/github/licensed"
+  url "https://github.com/github/licensed.git",
+      tag:      "3.7.3",
+      revision: "76727f75d486a24758890a030e540ebf87bba78b"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ruby@2.7"
```

Comment:
any issue tracker for supporting ruby@3?

Reply:
@chenrui333 

I change to ruby@3 because I can't find any issues with ruby@3 on the repository and I find the CI with ruby@3.

https://github.com/github/licensed/blob/a4a0534196a9998d333b1364653e5dd7fb9bcf52/.github/workflows/test.yml#L180

I used ruby@2.7, based on the .ruby-version of the repository .

https://github.com/github/licensed/blob/master/.ruby-version